test/parallel/test-runner-output.mjs

```diff
@@ -219,6 +219,26 @@ const tests = [
     name: 'test-runner/output/test-runner-plan.js',
     flags: ['--test-reporter=tap'],
   },
+  {
```

Comment:
Can we put these in a new test file. This one is already a bit of a nightmare to debug.